### PR TITLE
Unathi Consular Weapon Tweaks

### PR DESCRIPTION
**Pull request by Geevies**
* Replaced the Izweski Hegemony Consular's Chinese Dao with a standard Hegemony Energy Pistol.
* Gave the above Consular a waist holster.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/background/citizenship/unathi.dm [hunk 1]
```diff
@@ -39,6 +39,7 @@
 	name = "Izweski Hegemony Consular Officer"
 
 	uniform = /obj/item/clothing/under/unathi
+	accessory = /obj/item/clothing/accessory/holster/waist
```
Comment (Alberyk) [inline code/modules/background/citizenship/unathi.dm]: All other consulars don't spawn with a holster, I see no reason to give the unathi one. Either they don't spawn with one, or all spawn with a holster. Plus, they can get it from the loadout.

Comment (Geevies) [inline code/modules/background/citizenship/unathi.dm]:
```suggestion
```

File: html/changelogs/geeves-unathi_consular.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - tweak: "Replaced the Izweski Hegemony Consular's Chinese Dao with a standard Hegemony Energy Pistol."
+  - rscadd: "Gave the above Consular a waist holster."
```
Comment (Geevies) [inline html/changelogs/geeves-unathi_consular.yml]:
```suggestion
```

**Pull request merged**